testing/trino-product-tests/src/main/java/io/trino/tests/hive/TestHiveViews.java

```diff
@@ -13,5 +13,71 @@
  */
 package io.trino.tests.hive;
 
+import io.trino.tempto.Requires;
+import io.trino.tempto.assertions.QueryAssert;
+import io.trino.tempto.fulfillment.table.hive.tpch.ImmutableTpchTablesRequirements;
+import org.testng.annotations.Test;
+
+import static io.trino.tempto.assertions.QueryAssert.Row.row;
+import static io.trino.tempto.assertions.QueryAssert.assertThat;
+import static io.trino.tempto.query.QueryExecutor.query;
+import static io.trino.tests.TestGroups.HIVE_VIEWS;
+import static io.trino.tests.utils.QueryExecutors.onHive;
+import static io.trino.tests.utils.QueryExecutors.onPresto;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+@Requires({
+        ImmutableTpchTablesRequirements.ImmutableNationTable.class,
+        ImmutableTpchTablesRequirements.ImmutableOrdersTable.class,
+})
```

Comment:
I very much dislike annotation-based requirements, because they make working with tests harder.
Anyway `TestHiveViewsLegacy` already uses this, so please keep it.

import the inner classes